### DISCUSSION TITLE
Next Meetup?

### DISCUSSION THREAD

**Discussion by Marco-Affinidi** [berlin-hack-and-tell/bhnt.c-base.org#337] [General]
Hey folks!

Isn't it about time to do another meetup?
Anything one could help with?

Best
Marco

**Comment by ligi** [comment]
Unfortunately we could not do the October one as I was in colombia for DEVCon and we found no other host for this month. I will be back for the November one. But more people on the team (especially people that are willing to host) are always welcome. Would love if we get it happen regularly every month again! Please approach us at the next BHNT if you could imagine to be part of the team and maybe host events.

**Comment by samzmann** [comment]
Sounds good. So we're talking late November for the next one? Probably Tuesday 29th (last Tuesday of the month)?
I'm down to help organize a few things, or at least have a chat about it. Will come see you at the next meetup. And I think I'll have a little hack to present too :)

Sam ✌️
